composer.json

```diff
@@ -14,11 +14,13 @@
     ],
     "homepage": "https://github.com/koninklijke-collective/my_user_management",
     "license": "LGPL-3.0+",
+    "require": {
+        "typo3/cms-core": "11.*@dev",
+        "typo3/cms-beuser": "11.*@dev"
+    },
     "require-dev": {
-        "typo3/cms-core": "10.*.*@dev",
-        "typo3/cms-beuser": "10.*.*@dev",
-        "typo3/cms-install": "10.*.*@dev",
-        "typo3/cms-about": "10.*.*@dev",
+        "typo3/cms-install": "11.*@dev"
```

Comment:
I wanted to use your version, but there's an error here. There's a comma missing at the end of the line.

Reply:
Updated and fixed.

Reply:
Great, thanks!